packages/connect-query/src/index.ts

```diff
@@ -12,23 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import type { Message } from "@bufbuild/protobuf";
+
+import type { UnaryHooks } from "./create-hooks";
+import type { UnaryFunctions } from "./create-unary-functions";
+
 export { createQueryService } from "./create-query-service";
 export type {
-  QueryHooks,
+  QueryFunctions,
```

Comment:
It looks like we're removing or changing several export, so we do make a breaking change here?

Reply:
Yeah sorry, i forgot to put this PR into draft. I will add more details to the PR. There are some types that have breaking changes (though fairly light, with a replacement type that does the same). I'll update the description.